css/main.css

```diff
@@ -1,3 +1,163 @@
-/* Add your custom CSS here */
+* {
+    font-family: 'Merriweather Sans', sans-serif;
+    margin: 0;
+    padding: 0;
+    line-height: 1.4rem;
 
-/* Dont' forget to link this file to your HTML in the <head> */
+}
+
+.logo {
+    height: 5rem;
+    width: 5rem;
+    padding: 1rem;
+}
+
+header {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: absolute;
+}
+
+ul {
+    padding: 1rem;
+}
+
+ul>li {
+    list-style: none;
+    display: inline-flex;
+}
+
+li>a {
+    text-decoration: none;
+    color: black;
+    padding-right: 1.5rem;
+}
+
+.hero {
+    /* text-align: center; */
+    background-image: url("../css/img/hero-house.jpeg");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    min-height: 80vh;
+
+}
+
+.hero>h2 {
+    padding-top: 150px;
+    text-align: center;
+
+}
+
+div>img {
+    max-width: 100%;
+}
+
+#houses>h2 {
```

Comment:
Is there a reason to use an ID (`#`) over a classname or something different here?